src/main/java/Duke.java

```diff
@@ -1,10 +1,120 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Class that runs the entire Duke application
+ */
 public class Duke {
+
+    private static final String FOLDER_PATH = "../src/main/java/data/";
+    private static final String FILE_NAME = "All Tasks.txt";
+
+    /**
+     * Runs the Duke application
+     */
+    public static void run() {
```

Comment:
This method is too long. Maybe you want to abstract something out and cut it down a bit.
According to the code quality chapter in the textbook, a method should not get more than 30 lines. 
You may want to check here: https://nus-cs2103-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html